src/mixins/filing-template-mixin.ts

```diff
@@ -262,7 +262,7 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     }
 
     // restore the amalgamation court approval
-    if (draftFiling.amalgamationApplication.courtApproval) {
+    if (draftFiling.amalgamationApplication.courtApproval != null) {
```

Comment:
I don't think you need to change this as the previous syntax does exactly this!

Reply:
could you plz explain more? cause I try without this change, when I select "without court approval", the selection cannot be saved

Reply:
OK, it's fine. But please if you can change it to `!==`. Thanks!

Reply:
I think `courtApproval` may be True or False, which is why the check for null is needed.

Reply:
Shaoyun, since this caused confusion once, and to avoid a future developer trying to "optimize" this and actually break it, please add a code comment explaining why you check for not null (ie, save value if it's True or False).

Otherwise, this code looks good to me.

Reply:
The updated comment is fine by me. Resolving.